internal/pkg/clock/chainclock.go

```diff
@@ -0,0 +1,42 @@
+package clock
```

Comment:
Spec for reference: https://filecoin-project.github.io/specs/#systems__filecoin_nodes__clock